src/3box.js

```diff
@@ -529,16 +515,51 @@ class Box extends BoxApi {
   }
 }
 
-function initIPFSRepo () {
+function createIframeCache () {
+  const iframe = document.createElement('iframe')
+
+  iframe.src = process.env.CACHE_IFRAME_URL
```

Comment:
The source should be hardcoded in the `config.js` module, with an option to change it.

Reply:
I did not catch `config.js`, thanks for pointing that out!